src/hooks/useMapClickHandlers.ts

```diff
@@ -72,11 +72,13 @@ export function useMapClickHandlers() {
   );
 
   const handleClusterClick = useCallback(
-    (markers: MarkerData[], count: number = 0) => {
+    (location: any, count: number = 0) => {
```

Comment:
i am going to hold off on this pr just because of this to not cause conflicts. i am in the process of removing the `MarkerData` type and my changes touch here.